OpenMarket/OpenMarket/Resource/Controller/ProductsViewController.swift

```diff
@@ -0,0 +1,207 @@
+        return Constant.edgeInsetValue
+    }
+    
+    func collectionView(
+        _ collectionView: UICollectionView,
+        layout collectionViewLayout: UICollectionViewLayout,
+        minimumInteritemSpacingForSectionAt section: Int
+    ) -> CGFloat {
+        return Constant.edgeInsetValue
+    }
+}
+
+// MARK: UICollectionViewDataSource
+extension ProductsViewController: UICollectionViewDataSource {
+    func collectionView(
+        _ collectionView: UICollectionView,
+        numberOfItemsInSection section: Int
+    ) -> Int {
+        guard let products = productsData?.products else { return 0 }
+        return products.count
+    }
+
+    func collectionView(
+        _ collectionView: UICollectionView,
+        cellForItemAt indexPath: IndexPath
+    ) -> UICollectionViewCell {
+        guard let products = productsData?.products else { return UICollectionViewCell() }
+        guard let cell = collectionView.dequeueReusableCell(
+            withReuseIdentifier: ProductItemCell.identifier,
+            for: indexPath
+        ) as? ProductItemCell else {
+            return UICollectionViewCell()
+        }
+        let product = products[indexPath.row]
+        
+        cell.configureLayout(index: segmentIndex)
+        cell.titleLabel.text = product.name
+        cell.setPriceLabel(
+            originPrice: product.originPriceStringValue,
+            bargainPrice: product.bargainPriceStringValue,
+            segment: segmentIndex
+        )
+        cell.setStockLabelValue(stock: product.stock)
+        
+        guard let url = URL(string: product.thumbnail) else { return cell }
+        let imageTask = URLSession.createTask(url: url) { image in
+            DispatchQueue.main.async {
+                cell.thumbnailImageView.image = image
+            }
+        }
+        
+        cell.task = imageTask
+        return cell
+    }
+}
+
+// MARK: Objc Method
+private extension ProductsViewController {
+    @objc func didChangedSegmentIndex(_ sender: UISegmentedControl) {
+        let index = sender.selectedSegmentIndex
+        
+        segmentIndex = index
+        collectionView.reloadData()
+    }
+    
+    @objc func didTappedAddButton() {
+        let viewController = UIViewController()
+        
+        viewController.view.backgroundColor = .systemPink
+        navigationController?.pushViewController(viewController, animated: true)
+    }
+}
+
+// MARK: Business Login
+private extension ProductsViewController {
+    func fetchData() {
+        let endPoint = OpenMarketAPI.productsList(pageNumber: currentPage, rowCount: 200)
+        productResponseNetworkManager.fetchData(endPoint: endPoint) { result in
+            switch result {
+            case .success(let data):
+                self.productsData = data
+            case .failure(let error):
+                print(error)
+            }
+        }
+    }
+}
+
+// MARK: Configure UI
+private extension ProductsViewController {
+    func configureNavbar() {
```

Comment:
```suggestion
    func configureNavigationbar() {
```

간결한 것 보다 명확한것이 중요합니다.